src/page-modules/assistant/layout.tsx

```diff
@@ -45,6 +50,7 @@ function AssistantLayout({ children, tripQuery }: AssistantLayoutProps) {
   const [isPerformingSearchNavigation, setIsPerformingSearchNavigation] =
     useState(false);
   const [geolocationError, setGeolocationError] = useState<string | null>(null);
+  const [defaultFilters, setDefaultFilters] = useState<string[] | null>(null);
```

Comment:
Do we need a separate state for this? Don't we just need to overwrite the state on the initial load, and can use the URL state to know whether it should be done or not?

Reply:
No, I do not think we need it. I added it because I when i tried to find out why the flight filter did not appear in preview and forgot to remove it. Will fix now👍